src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -770,9 +770,12 @@ private static String processPageSourceForGodMode(String content) {
         return content
                 .replaceAll("<#comment[ ]*</#comment>", "<!---->")
                 .replace(Config.APP_URL, "${app.url}")
-                .replace("http://localhost:8888", "${web.url}")
-                .replace(Config.APP_URL.replace("http", "https"), "${web.url}")
-                .replace("\"/_ah", "\"${web.url}/_ah")
+                .replace(TestProperties.inst().TEAMMATES_URL, "${test.url}")
+                .replace(TestProperties.inst().TEAMMATES_URL.replace("http", "https"), "${test.url}")
+                // this is to handle dev server case where url is relative
+                .replace("\"/_ah", "\"${test.url}/_ah")
+                // this handles the logout url that google generates
+                .replaceAll("_ah/logout\\?continue=.*?\"", "_ah/logout?continue={*}\"")
```

Comment:
Can we combine these two? Something like
`.replaceAll("\"/_ah/logout\\?continue=.*?\"", "\"${test.url}/_ah/logout?continue={*}\"")`.